pyqtgraph/graphicsItems/ImageItem.py

```diff
@@ -640,7 +640,7 @@ def _try_rescale_float(self, image, levels, lut):
 
         minVal, maxVal = levels
         if minVal == maxVal:
-            maxVal = xp.nextafter(maxVal, 2*maxVal)
+            maxVal = xp.nextafter(maxVal, 2*maxVal) if maxVal != 0 else numpy.finfo(maxVal).eps
```

Comment:
I have a dumb question:
Is it actually intentional that the existing code moves `maxval` to the next more *negative* number when `maxval` is already < 0? That makes `rng` negative below.

If the intention is to clamp `rng` to a small positive number, then wouldn't
```maxVal = math.nextafter( minVal, math.inf )```
be what we want?